src/nss/inc_treasure.nss

```diff
@@ -27,6 +27,8 @@ const int WAND_VALUE_MULTIPLIER = 2;
 const int SCROLL_VALUE_MULTIPLIER = 6;
 const int MISC_VALUE_MULTIPLIER = 5;
 
+const string MAGIC_THROWING_WEAPON_DESCRIPTION = "\n\nEnchanted throwing weapons return to the wielder's hand when thrown.";
```

Comment:
"Enchanted" might be the wrong word, because "High Quality" weapons are affected too.
I don't have a better idea, though!

Reply:
We've removed high quality weapons, unfortunately. In any case they wouldnt have been included anyways